### ISSUE TITLE
Each sentence should have its own line

### ISSUE THREAD

**Issue by ianzapolsky** [why-el/crowd-written-novel#16]
Conforming to this standard will make reading commit diffs much easier.

**Comment by Ghoughpteighbteau** [comment]
yes! I know this is unusual, and we can add a script in to merge paragraphs back together as a build phase, but git does need new lines to be conceptually separate pieces. A limitation of the tool!

**Comment by n-r** [comment]
Markdown automatically merges paragraphs back together. By the way, I also support the idea of each sentence having its own line and have changed my pull request accordingly.

**Comment by Ghoughpteighbteau** [comment]
Oh damn you're right, we can take advantage of markdown! Excellent point n-r.
